src/template.rs

```diff
@@ -10,6 +10,9 @@ pub struct Template<'a> {
 impl<'a> Renderable for Template<'a> {
     fn render(&self, context: &mut Context) -> Result<Option<String>> {
         context.filters.insert("size".to_string(), Box::new(size));
+        context.filters.insert("upcase".to_string(), Box::new(upcase));
+        context.filters.insert("minus".to_string(), Box::new(minus));
+        context.filters.insert("replace".to_string(), Box::new(replace));
```

Comment:
We  really need to find a better way to do this than create everything at render eventually, but that's not for this PR :sweat_smile: